### PR TITLE
Added Babylonian method in Go, Java

### LINKED ISSUE #1063: Adding Babylonian method in Java and Go
Please assign it to me. It is not implemented in these languages.

### PR DESCRIPTION
**Pull request by MastersAbh**
Fixes #1063  

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

- Added Babylonian method in Java and Golang

#### Languages Used:
- Java
- Go

#### Files Added:
- Babylonian_Method.java
- Babylonian_Method.go

### CODE REVIEW

File: Babylonian_Method/Babylonian_Method.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/* Babylonian method for square root
+  1. Start with an arbitrary positive start value x (the closer to the 
+     root, the better).
+  2. Initialize y = 1.
+  3. Do following until desired approximation is achieved.
+      a) Get the next approximation for root using average of x and y
+      b) Set y = n/x */
+
+package main
```
Comment (tsah20) [inline Babylonian_Method/Babylonian_Method.go]: Please change the package name to something meaningful

File: Babylonian_Method/Babylonian_Method.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    for (number - guess > accuracy) {
+        number = (number + guess) / 2 
+        guess = num / number // to get next approximation 
+    } 
+    return number; 
+} 
+
+func main(){
+    var num float64 = 2
+    fmt.Print("Square root of " + strconv.Itoa(int(num)) + " is " +  fmt.Sprintf("%f",squareRoot(num)))
```
Comment (somya-kapoor) [inline Babylonian_Method/Babylonian_Method.go]: Please add space before squareRoot(num)

File: Babylonian_Method/Babylonian_Method.go (hunk 1 above)
Comment (MastersAbh) [reply Babylonian_Method/Babylonian_Method.go]: In Go, we have package main as the root package, like it is int main() in C++.,
So I guess it makes sense.

**Comment by MastersAbh** [comment]
Done, Please review @tsah20

**Comment by somya-kapoor** [comment]
@MastersAbh Good work 
@jainaman224 This pr is approved, please have a look.

**Pull request merged**